components/ui-shell/index.scss

```diff
@@ -293,7 +293,7 @@ $prefix: "bx";
 
   #{$prefix}-side-nav-items {
     overflow-y: scroll;
-    padding: var(--cds-spacing-03) 0;
+    padding: 0;
```

Comment:
@eddybrando @Eric-Arellano 

if you see a need for this spacing, we can keep it... otherwise, when that first item is active and therefore shows a border, it looks weird

Reply:
No, sounds good to me to get rid of the spacing!